### DISCUSSION TITLE
change default tile size of WMS request in Hajk

### DISCUSSION THREAD

**Discussion by bitmapbulgaria** [hajkmap/Hajk#1435] [Q&A] [answered]
Hi! 
When we have huge monitor resolution (main is 34440 to 1140), the large area and 256 size of a tile is a problem these days- the Geoserver have to generate huge amount of tiles to serve the GWC content. 
A tile size of 512x512 pixels is more suitable. In the modern world we have fast internet and large monitor...
Does somebody know how to change the default size of requests (for example - to WMS layers in Hajk from 256 to 512 pixels? 
I have done this once, but I forgot and now I'm stuck.
P.S. Maybe the 512 pixels default tile size is good to be default in next versions in Hajk. 
Seems reasonable for me...

**Comment by jacobwod** [comment, marked as answer]
It has to do with your WMS settings, especially the Gridset you define, but it's surely doable in Hajk today. The screenshot below is from the public https://karta.halmstad.se when device requests HiDPI tiles (because of the device's pixel ratio).

![image](https://github.com/hajkmap/Hajk/assets/110222/758bce0b-e6f0-4d63-8657-f2c01cfd2222)

**Reply by bitmapbulgaria** [reply]
Thanks for sharing this!
Update: 
I found my initial solution (not so flexible, but simple and easy to set):
Just set the DEFAULT_TILE_SIZE = 512 (or whatever is needed) in /new-client/node_modules/ol/tilegrid/common.js
The build will get this size for every request from now on.
Pros: Very easy. Not necessary to setup Server FORMAT OPTIONS for the requests.
Cons: In production, this is not possible to change ( even if manually change "512, 512" in all .js chunks...
